src/components/app/ProfileFilterNavigator.js

```diff
@@ -70,8 +70,18 @@ class ProfileFilterNavigatorBarImpl extends React.PureComponent<Props> {
       );
     } else {
       firstItem = (
-        <Localized id="ProfileFilterNavigator--full-range">
-          Full Range
+        <Localized
+          id="ProfileFilterNavigator--full-range-with-duration"
+          vars={{
+            fullRangeDuration: getFormattedTimeLength(
+              rootRange.end - rootRange.start
+            ),
+          }}
+        >
+          <>
+            Full Range (
+            {getFormattedTimeLength(rootRange.end - rootRange.start)})
```

Comment:
What do you think of not using `({getFormattedTimeLength(rootRange.end - rootRange.start)})` here, since it will be replaced by Fluent anyway, it seems counterproductive to run the full process in this case.